readme.rst

```diff
@@ -620,14 +620,75 @@ Function Reference:
 
 `fix_variables(bqm, sampling_mode=True) <https://docs.ocean.dwavesys.com/projects/dimod/en/latest/reference/bqm/generated/dimod.roof_duality.fix_variables.html?highlight=fix_variables#dimod.roof_duality.fix_variables>`_
 
+Other Tools
+===============================
+
+qbsolv
+-----------------
+
+The ``qbsolv`` utility has for all intents and purposes been replaced with the ``dwave-hybrid`` framework in Ocean
```

Comment:
This is not accurate. I think it would be more accurate to say "qbsolv's functionality has been folded into a more general framework" or similar.

Reply:
haha ok. i will reword this.